config/initializers/01_toxiproxy.rb

```diff
@@ -0,0 +1,18 @@
+class Toxiproxy
```

Comment:
There should be one config file for both then instead of relying on how Rails loads files.